CHANGELOG.md

```diff
@@ -1,22 +1,32 @@
 # Change Log
 
 ## v1.3.0
+
+- [New] Add `direction` option (prop) to `withDirection` HOC, This to allow the overwrite of the direction when required (for phone numbers for example that are always LTR)
+- [Dev Deps] update `brcast`, `deepmerge`, `hoist-non-react-statics`, `@babel/cli`, `@babel/core`, `babel-jest`, `babel-plugin-transform-replace-object-assign`, `babel-preset-airbnb`, `eslint`, `jest`, `sinon`, `webpack`
+
+## v1.3.0
+
 - [New] Add `AutoDirectionProvider` component for automatically detected direction.
 
 ## v1.2.0
+
 - [New] Add `inline` option; if specified, will render a `span` instead of a `div`.
 
 ## v1.1.0
+
 - [New] increase peerDep/devDep range for react and bump enzyme (#2)
 - [Deps] update `hoist-non-react-statics`, `prop-types`
 - [Dev Deps] update `babel-jest`, `chai`, `eslint`, `eslint-config-airbnb`, `eslint-plugin-jsx-a11y`, `eslint-plugin-react`, `jest`, `rimraf`, `sinon`, `sinon-chai`, `webpack`
 
 ## v1.0.1
- - [Fix] allow down to React v15.0
- - [Deps] update `airbnb-prop-types`, `deepmerge`, `hoist-non-react-statics`
- - [Dev Deps] update `babel-cli`, `casual`, `eslint`, `eslint-plugin-jsx-a11y`, `eslint-plugin-react`, `sinon-chai`, `sinon`, `webpack`; remove `eslint-config-airbnb-base`
- - [Dev Deps] add `babel-jest`, which needs to be explicitly a top-level dev dep for npm 2
- - [Tests] test on node 4, but not node 7
+
+- [Fix] allow down to React v15.0
```

Comment:
the space is intended; the bug is that later sections *lack* a space.

Reply:
I reverted the changes